docs/Use Case.md

```diff
@@ -0,0 +1,22 @@
+# Use Case
+
+**Title:** Determining the health of a Github repository  
+
+**Primary Actor:** Project Leader 
+
+**Goal in Context:** The project lead is able to access if an open source project is healthly enough for its company's standards to incorporate into their product.  
+
+**Stakeholders:**  
+* Project Lead: To understand if software is apporiate to incorporate into their own product
+* Developer: To clearly gauge if a piece of software will need to be maintained by them or if it's reliable
+* Manager: To access the longevity of this project with regards to dependencies
+
+**Preconditions:**  
+* Metrics for a healthy repository are already defined
+* The repositories to be compared are already picked out 
+
+**Main Success Scenario:** Project lead understands the metrics presented and makes an accurate decision on if the software should be incoporated  
+
+**Failed End Conditions:** Project lead recieves little to no information on the health of the chosen repository.  
```

Comment:
Nitpick: Some of these are bullets and some are sentences. Consistently use periods at the end of the sentences and bullets can go either way as long as they are consistent.